channels/backends/redis_py.py

```diff
@@ -64,7 +63,11 @@ def connection(self, index):
     def send(self, channel, message):
         # if channel is no str (=> bytes) convert it
         if not isinstance(channel, str):
-            channel = channel.decode('utf-8')
+            channel = channel.decode("utf-8")
+        # Write out message into expiring key (avoids big items in list)
+        # TODO: Use extended set, drop support for older redis?
+        key = self.prefix + uuid.uuid4().hex
```

Comment:
I moved the definition of `key` up here, as it doesn't seem like it's defined when `index = self.consistent_hash(key)` is called here: https://github.com/andrewgodwin/channels/blob/master/channels/backends/redis_py.py#L71